### PR TITLE
switch back to app-local VC runtimes and UCRT

### PR DESCRIPTION
**Pull request by birarda**
In an effort to remove a step from the Steam and native installers that has frustrated and confused users in testing, I was investigating a return to app-local VC runtimes. I found that CMake 3.9.2 contained a fix for InstallRequiredSystemLibraries with VS 2017. I have tested the below with CMake 3.10.0 and confirmed it places the expected UCRT DLLs and VC runtime DLLs in the install directory.

- include the universal CRT to app-locally support older windows versions (per https://blogs.msdn.microsoft.com/vcblog/2015/03/03/introducing-the-universal-crt/)
- go back to using InstallRequiredSystemLibraries to place needed VC runtimes app-locally
- tell windeployqt not to grab any compiler runtimes

### CODE REVIEW

File: cmake/macros/PackageLibrariesForDeployment.cmake [hunk 1]
```diff
@@ -39,7 +39,7 @@ macro(PACKAGE_LIBRARIES_FOR_DEPLOYMENT)
     add_custom_command(
       TARGET ${TARGET_NAME}
       POST_BUILD
-      COMMAND CMD /C "SET PATH=%PATH%;${QT_DIR}/bin && ${WINDEPLOYQT_COMMAND} ${EXTRA_DEPLOY_OPTIONS} $<$<OR:$<CONFIG:Release>,$<CONFIG:MinSizeRel>,$<CONFIG:RelWithDebInfo>>:--release> \"$<TARGET_FILE:${TARGET_NAME}>\""
+      COMMAND CMD /C "SET PATH=%PATH%;${QT_DIR}/bin && ${WINDEPLOYQT_COMMAND} ${EXTRA_DEPLOY_OPTIONS} $<$<OR:$<CONFIG:Release>,$<CONFIG:MinSizeRel>,$<CONFIG:RelWithDebInfo>>:--release> --no-compiler-runtime \"$<TARGET_FILE:${TARGET_NAME}>\""
```
Comment (jherico) [inline cmake/macros/PackageLibrariesForDeployment.cmake]: Looking at the options for windeployqt, we should also add `--no-opengl-sw`, `--no-angle` `-no-system-d3d-compiler` to eliminate some of the cruft we're getting (we don't rely on the GLES over D3D implementation that angle provides, or the software GL implementation, or d3d at all)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7598/HighFidelity-Beta-PR12524-e1421f2e3afb7a9f682fdb790a66e9bbdac2805e.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7598/HighFidelity-Beta-PR12524-e1421f2e3afb7a9f682fdb790a66e9bbdac2805e.dmg

File: cmake/macros/PackageLibrariesForDeployment.cmake (hunk 1 above)
Comment (birarda) [reply cmake/macros/PackageLibrariesForDeployment.cmake]: 👍

**Comment by birarda** [comment]
Keeping this DNM until we confirm an ops system for updating the VC runtime when desired, and that build boxes are on CMake 3.1.0

**Comment by birarda** [comment]
Need to double check that the multi-line format used for windeployqt doesn't screw up the command.

Also need to verify on a fresh install that the expected DLLs are present.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7611/app_12524_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7611/HighFidelity-Beta-PR12524-0d7d382540f7316af7ab9d61503500fd85be09d4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7611/HighFidelity-Beta-PR12524-0d7d382540f7316af7ab9d61503500fd85be09d4.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7889/app_12524_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7889/HighFidelity-Beta-PR12524-2158acecf4941be566cd599f35a4d69d75989545.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7889/HighFidelity-Beta-PR12524-2158acecf4941be566cd599f35a4d69d75989545.dmg

**Comment by birarda** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8044/app_12524_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8044/HighFidelity-Beta-PR12524-2158acecf4941be566cd599f35a4d69d75989545.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8044/HighFidelity-Beta-PR12524-2158acecf4941be566cd599f35a4d69d75989545.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8050/app_12524_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8050/HighFidelity-Beta-PR12524-10dd16ae8d53ff0216cb50a4f4a9314ac6d78ff5.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8050/HighFidelity-Beta-PR12524-10dd16ae8d53ff0216cb50a4f4a9314ac6d78ff5.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8059/app_12524_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8060/app_12524_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8060/HighFidelity-Beta-PR12524-499defe77ea061b4148c94abeca0e8d26088506d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8060/HighFidelity-Beta-PR12524-499defe77ea061b4148c94abeca0e8d26088506d.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8065/app_12524_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8066/app_12524_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8066/HighFidelity-Beta-PR12524-f8df90e36d64a27bdf1d3a90c60bc61b84e372e8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8066/HighFidelity-Beta-PR12524-f8df90e36d64a27bdf1d3a90c60bc61b84e372e8.exe

**Comment by birarda** [comment]
#### Test Plan

1. Download the installer from this PR and run it. Ensure that Interface and the Sandbox launch and run without crashing.
2. Confirm that the install folder (which should be `C:\Program Files\High Fidelity - PR12524`) contains `msvcp140.dll, concrt140.dll, vcruntime140.dll, msvcp120.dll, msvcr120.dll` and does not contain `sixense_x64.dll` or `plugins/hifiSixense.dll`.
3. Download this tool (https://github.com/lucasg/Dependencies/releases/download/v1.6/Dependencies.zip) and open the installed Interface.exe from the PR folder.
4. Confirm that the Dependencies program shows that it has loaded `MSVCP140.dll` and `VCRUNTIME140.dll` from your install folder and not from the system folder
5. Open `Leap.dll` from your install folder with Dependencies. Confirm that it shows that it has loaded `MSVCP120.dll` and `MSVCR120.dll` from your install folder and not from the system folder.

**Comment by birarda** [comment]
Marking DO NOT MERGE since ops will need to confirm that they have installed the latest VS 2013 redistributables on all build boxes (cc: @marko8904).

File: CMakeLists.txt [hunk 2]
```diff
@@ -148,16 +146,18 @@ if (BUILD_SERVER)
...
 if (BUILD_CLIENT)
-    add_subdirectory(interface)
-    set_target_properties(interface PROPERTIES FOLDER "Apps")
-    if (ANDROID)
-        add_subdirectory(gvr-interface)
-        set_target_properties(gvr-interface PROPERTIES FOLDER "Apps")
-    endif()
+  add_subdirectory(interface)
+  set_target_properties(interface PROPERTIES FOLDER "Apps")
+  if (ANDROID)
```
Comment (jherico) [inline CMakeLists.txt]: this code block is no longer needed.  Note that the `if (ANDROID)` block above does an early exit from this cmake file.

File: cmake/templates/NSIS.template.in [hunk 3]
```diff
@@ -858,12 +855,6 @@ Section "-Core installation"
   ; Rename the incorrectly cased Raleway font
   Rename "$INSTDIR\resources\qml\styles-uit\RalewaySemibold.qml" "$INSTDIR\resources\qml\styles-uit\RalewaySemiBold.qml"
 
-  ExecWait "$INSTDIR\vcredist_x64.exe /install /q /norestart"
-
-  ; Remove the Old Interface directory and vcredist_x64.exe (from installs prior to Server Console)
-  RMDir /r "$INSTDIR\Interface"
```
Comment (jherico) [inline cmake/templates/NSIS.template.in]: why are we removing this line?  It doesn't seem related to the runtime changes.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8077/app_12524_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8077/HighFidelity-Beta-PR12524-283eccd043bb6b3598adc518771f94c2ad0fce92.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8077/HighFidelity-Beta-PR12524-283eccd043bb6b3598adc518771f94c2ad0fce92.exe

**Comment by qlimnavarro** [comment]
Currently Testing.

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
MSVCP120.dll and MSVCR120.dll appear in the Dependencies app. Interface and Sandbox run as expected.

**Comment by birarda** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8161/app_12524_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8161/HighFidelity-Beta-PR12524-283eccd043bb6b3598adc518771f94c2ad0fce92.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8161/HighFidelity-Beta-PR12524-283eccd043bb6b3598adc518771f94c2ad0fce92.exe

**Comment by birarda** [comment]
Double checked with ops that VS2013 was installed on all windows build boxes - downloaded the latest installer and confirmed that msvcp120 and msvcr120 were present in the archive
